users/models.py

```diff
@@ -100,3 +102,18 @@ def soft_delete(self: "User"):
         self.posts.update(curation_status=Post.CurationStatus.DELETED)
 
         self.save()
+
+        send_email_async.send(
+            subject="Your Metaculus Account Has Been Deactivated",
+            message=textwrap.dedent(
+                """Your Metaculus account has been deactivated by an administrator or an automated system. Possible reasons could include
+                - Suspicious activity
+                - Spam/Ad/Inappropriate content in comments
+                - Spam/Ad/Inappropriate content in profile bio
+                - Manual review for bot and spam accounts
+
+                If you believe this was done in error, please contact support@metaculus.com and we will reactivate your account."""
+            ),
+            from_email=settings.EMAIL_HOST_USER,
+            recipient_list=[self.email],
+        )
```

Comment:
I think we should keep models isolated from business logic like emails sending, interacting with brokers and etc. I'd propose to create a `soft_delete_user` service in `users.services` and put this logic there.

Reply:
I could go either way on it triggering here.
This means an admin soft deleting users from the admin panel would trigger this. I'm not sure if that's desired.
Then again, soft deleting a user inappropriately should probably give them that info.